### ISSUE TITLE
Incorrect incognito detection

### ISSUE THREAD

**Issue by skywinder** [austintgriffith/burner-wallet#244]
# Bounty

## Scope

- Incorrect incognito detection: 
- After several minutes (3-7) burner wallet changes from normal mode to incognito mode. Actually, it is not incognito.

### Steps to reproduce: 
Open Firefox in normal mode
1. go to xdai.io
1.  wait for 5-10 minutes. 
1. Burner shows,  that you are in incognito mode.

**- OS: iOS 12 / iPhone XR**
**- Browser: Firefox 18.1**

## Deliverables
- We have to check code in this scope: https://github.com/austintgriffith/burner-wallet/blob/77a0b0dc9a8e475e54ef71e2924ed3d87f71671c/src/services/incogDetect.js#L2-L20

## Roles
bounty gardener: @skywinder / 10%
bounty worker: name / 75%
bounty reviewer: name / 15%

![image](https://user-images.githubusercontent.com/3356474/62954656-72f97180-bdf8-11e9-944b-0c52bde414a6.png)

![image](https://user-images.githubusercontent.com/3356474/62954699-83a9e780-bdf8-11e9-942b-fa526b46e269.png)

**Comment by yanyishuai** [comment]
Claiming #244 — focused fix per bounty scope.

**Comment by rovidev95** [comment]
False-positive incognito fix pushed to https://github.com/austintgriffith/burner-wallet/pull/317 — removed aggressive storage-quota heuristic; Firefox uses IndexedDB probe with 500ms timeout; inconclusive defaults to normal browsing. Fixes #244. @TimDaub
